web_src/js/features/codeeditor.js

```diff
@@ -156,6 +166,8 @@ export async function createCodeEditor(textarea, filenameInput, previewFileModes
 
   filenameInput.addEventListener('keyup', () => {
     const filename = filenameInput.value;
+    const _isMarkdown = markdownExts.includes(extname(filename));
```

Comment:
Underline prefix?

Reply:
This is to dricriminate from the `isMarkdown` used on page load above. What do you recommend for the name?

Reply:
I guess you could just move the `togglePreviewDisplay` call into `updateEditor`, `updateEditor` has full information about current filename

Reply:
Update: if it's not suitable to move the code to `updateEditor` (which is for editor only), the variable could have a new accurate name, for example: `isNewFilenameMarkdown`, etc

Reply:
> I guess you could just move the `togglePreviewDisplay` call into `updateEditor`, `updateEditor` has full information about current filename
> 
> Update: if it's not suitable to move the code to `updateEditor` (which is for editor only), the variable could have a new accurate name, for example: `isNewFilenameMarkdown`,
> 
> Or, just use `isMarkdown`, because there are also 2 `filename` variables.

Updated to use `isMarkdown`, because they have different scope so should be good. And I didn't move it to `updateEditor` because it seems `updateEditor` is for editor configuration update.